core-bundle/src/ServiceAnnotation/AbstractFragmentAnnotation.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\ServiceAnnotation;
+
+use Terminal42\ServiceAnnotationBundle\Annotation\ServiceTagInterface;
+
+abstract class AbstractFragmentAnnotation implements ServiceTagInterface
+{
+    /**
+     * @var array
+     */
+    private $attributes;
+
+    public function __construct(array $attributes)
+    {
+        if (isset($attributes['value'])) {
+            $attributes['type'] = $attributes['value'];
```

Comment:
What exactly does this do?

Reply:
*value* is the "first" attribute without name.

`@FrontendModule("two_factor", template="mod_one_factor")`

Will set the frontend module type to "two_factor" but use template "mod_one_factor".

Reply:
Nice. 😎